packages/react-emotion/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "babel-plugin-emotion": "^8.0.10",
+    "emotion-theming": "^8.0.10",
```

Comment:
This also needs to be added to `preact-emotion`. Also, in the rollup config, in the `preact-emotion` bit, `prop-types` and `emotion-theming` need to added to `external`.

Reply:
done